### PR TITLE
Subset loading samples from airtable for SVs and MITO

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Review by jklugherz — APPROVED** [review]

File: seqr/views/apis/data_manager_api_tests.py [hunk 1]
```diff
@@ -1654,12 +1666,18 @@ def _assert_write_pedigree_error(self, response):
 class AnvilDataManagerAPITest(AirflowTestCase, DataManagerAPITest):
     fixtures = ['users', 'social_auth', '1kg_project', 'reference_data']
 
+    ADDITIONAL_REQUEST_COUNT = 1
```
Comment (jklugherz) [inline seqr/views/apis/data_manager_api_tests.py:1669]: why is this needed?

Comment (hanars) [reply seqr/views/apis/data_manager_api_tests.py:1669]: The `AirflowTestCase` has some class level attributes it uses in its helper methods for validating that the calls to airflow are as expected. This is neccessary so that when it counts that there are the expected number of airflow calls it also knows that there is 1 additional request made thats not to airflow (namely the dag update call)
